packaging/dependencies/amazon-cloudwatch-agent-ctl

```diff
@@ -50,16 +51,17 @@ UsageString="
         3. query agent status:
             amazon-cloudwatch-agent-ctl -a status
 
-        -a: action
+        -a/--action: action
             stop:                                   stop the agent process.
             start:                                  start the agent process.
             status:                                 get the status of the agent process.
+            validate:                               validate the agent's configuration
             fetch-config:                           apply config for agent, followed by -c or -o or both. Target config can be based on location (ssm parameter store name, file name), or 'default'.
             append-config:                          append json config with the existing json configs if any, followed by -c. Target config can be based on the location (ssm parameter store name, file name), or 'default'.
             remove-config:                          remove config for agent, followed by -c or -o or both. Target config can be based on the location (ssm parameter store name, file name), or 'all'.
             set-log-level:                          sets the log level, followed by -l to provide the level in all caps.
 
-        -m: mode
+        -m/--mode: mode
```

Comment:
Doing this as a separate review so I only block on this.

---
I am hesitant on including a change that allows us to use long-name flags for existing functionality. I'm mostly concerned of scope creep, because the original ask was to add a new flag for just forcing strict validation. I'm going back on my suggestion/request earlier on using `-strict` long form, now that I see what is required to make that work, but I want others to weigh in on this.

Do we benefit enough from including this change as part of the PR, or should should we (1) change the strict validation flag back to just a single char flag and (2) back out this specific change and put it in a new issue to address separately?

Reply:
I would agree on add a single char flag for `strict` and add a long form for it though. You can see it in my last reply https://github.com/aws/amazon-cloudwatch-agent/pull/412#discussion_r835402859